pontoon/api/schema.py

```diff
@@ -176,5 +235,23 @@ def resolve_locale(obj, info, code):
 
         return qs.get(code=code)
 
+    def resolve_user(obj, info, username):
+        return UserModel.objects.get(username=username)
+
+    def resolve_translations(obj, info, project, locale):
+        qs = TranslationModel.objects
+        fields = get_fields(info)
+
+        if "user" in fields:
+            qs = qs.prefetch_related("translation__user")
+
+        if "locale" in fields:
+            qs = qs.prefetch_related("translation__locale")
+
+        return qs.filter(locale__code=locale, entity__resource__project__slug=project,)
```

Comment:
I think this should forbid the following cyclic queries:

    translations.user.actions.performed_by
    translations.locale.localizations.project.localizations
        (or remove Translation.locale)

Again, I assume this one can be called valid?

    translations.user.actions.translation

"Show me the translations which the author of each translation on the list somehow modified."